stubs/tensorflow/tensorflow/__init__.pyi

```diff
@@ -11,7 +11,42 @@ import numpy
 from tensorflow.dtypes import *
 
 # Most tf.math functions are exported as tf, but sadly not all are.
-from tensorflow.math import abs as abs
+from tensorflow.math import (
```

Comment:
Commenting out this import fixes pytype crash. I'm unsure what actual fix for it though is.

Reply:
What happens if you comment out just the `pow as pow` line?

Reply:
Tested and same error. Not specific to pow. `from tensorflow.math import add as add` by itself is enough to trigger crash.

Playing with it I found a workaround. I think the error is caused by import cycles and using wildcard import to define DType. If I explicitly import DType from tensorflow.dtypes the crash is fixed. @rchen152

Reply:
Thanks! For context: I wondered if `pow` might be the problem because of the fact that pytype uses a custom `builtins` stub rather than the one in typeshed -- this can often cause surprising errors in typeshed's CI. But great that you found the actual problem here, and a workaround :)

Reply:
Odd, typeshed definitely uses `import *` elsewhere, so I wonder why pytype was unhappy here. I'll look into it.